pkg/blockatlas/lending.go

```diff
@@ -0,0 +1,71 @@
+package blockatlas
+
+type ProviderType string
+
+const (
+	ProviderTypeLending ProviderType = "lending"
+	ProviderTypeStaking ProviderType = "staking"
+)
+
+type (
+	// LendingProvider static info about the lending provider, such as name and asset classes supported.
+	LendingProvider struct {
+		ID     string              `json:"id"`
+		Info   LendingProviderInfo `json:"info"`
+		Type   ProviderType        `json:"type"`
+		Assets []AssetInfo         `json:"assets"`
+	}
+
+	// LendingProviderInfo basic information about a lending provider.
```

Comment:
Why not reuse: https://github.com/trustwallet/blockatlas/blob/8e00b7387c40799e5a3f9654875c2947f6aac881/pkg/blockatlas/staking.go#L55 ?

Reply:
Done.